hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -114,15 +121,31 @@ public void doStateTransition() {
 				codeWithConstructorArgs,
 				txnCtx.consensusTime()
 		);
+
+		HederaAccountCustomizer customizer = null;
```

Comment:
If we are always passing the customizer, we can refactor the transition logic so that it is more clear, such as:
```Java
long expiry = RequestBuilder.getExpirationTime(txnCtx.consensusTime(), op.getAutoRenewPeriod()).getSeconds();
final var customizer = new HederaAccountCustomizer()
		.key(key)
		.autoRenewPeriod(op.getAutoRenewPeriod().getSeconds())
		.memo(op.getMemo())
		.expiry(expiry);

/* --- Persist changes into state --- */
final var createdContracts = worldState.persist(customizer);
if (result.isSuccessful()) {
	hederaLedger.customize(EntityIdUtils.asAccount(newContractId), customizer);
} else {
	worldState.reclaimContractId();
}
```